next/13/replace-next-head/index.ts

```diff
@@ -1372,8 +1372,10 @@ const getTsCompilerOptions = async (api: FileAPI, baseUrl: string) => {
 
 	try {
 		const tsConfigStr = await api.readFile(tsConfigPath);
-		return JSON.parse(tsConfigStr).compilerOptions;
+		const configWithoutComments = tsConfigStr.replace(/^\s*?\/\/.*$/gm, '');
```

Comment:
did you build this regex?

Reply:
yes. is it incorrect? it should match any inline comments

Reply:
I am concerned only about edge cases (matching comments within the JSON), but I think we can live with what we have now.